uninstall.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Undo Step 3: Removing copied config and local folders
+echo 'Removing copied config and local folders...'
+rm -rf "$HOME/.config"
+rm -rf "$HOME/.local"
+
+# Undo Step 2: Uninstall AGS
+echo 'Uninstalling AGS...'
+sudo meson uninstall -C ~/ags/build
+rm -rf ~/ags
+
+# Undo Step 1: Remove added user from video and input groups and remove yay packages
+echo 'Removing user from video and input groups and removing packages...'
+user=$(whoami)
+sudo deluser "$user" video
+sudo deluser "$user" input
+echo 'User rmoved from video and input groups.'
+
+# Removing installed yay packages and dependencies
+
+yay -Rns brightnessctl coreutils curl fish foot fuzzel gjs gnome-bluetooth-3.0 gnome-control-center gnome-keyring gobject-introspection grim gtk3 gtk-layer-shell libdbusmenu-gtk3 meson networkmanager nlohmann-json npm plasma-browser-integration playerctl polkit-gnome python-pywal ripgrep sassc slurp starship swayidle swaylock typescript upower xorg-xrandr webp-pixbuf-loader wget wireplumber wl-clipboard tesseract yad ydotool adw-gtk3-git cava gojq gradience-git gtklock gtklock-playerctl-module gtklock-powerbar-module gtklock-userinfo-module hyprland-git lexend-fonts-git python-material-color-utilities python-pywal python-poetry python-build python-pillow swww ttf-material-symbols-variable-git ttf-space-mono-nerd ttf-jetbrains-mono-nerd wlogout

```

Comment:
i'm having doubts... this is kinda destructive
i don't think it's a good idea to remove `coreutils upower ripgrep wget`
also remove nlohmann json please, this is no longer needed but i forgot to remove

Reply:
Makes sense, some of the packages are included with a gnome install after all

I'll get back to this after testing it on a system with hyprland installed

Reply:
I've removed arch-crucial packages and nlohmann-json, after comparing the vanilla arch packages installed with the packages to the removed above (I've removed wireplumber from the list to be uninstalled as well as it's usually installed with pipewire)